### ISSUE TITLE
HTTP Request integrated in Chatwoot

### ISSUE THREAD

**Issue by guibarbas** [chatwoot/chatwoot#9162]
### Is your feature or enhancement related to a problem? Please describe.

No, it’s just a feature request

### Describe the solution you'd like

It would be really useful to have within the automations an option of type HTTP Request.

Similar to what we have available in other tools like: N8N or Typebot.

With simple functions such as:
Method: POST, GET, PUT and etc.
Header.
Body
and other custom fields.

This would bring a lot of flexibility to the system, because it would allow us, through automations or even macros, to perform requests and queries in API’s, interact with systems and even "Pause Typebot" sending a simple "pause" through the api, when the bot is no longer needed, without the need for external systems such as N8N

### Describe alternatives you've considered

Currently, we use the N8N to supply this feature, and automation by sending a call via Webhook, however, having an internal HTTP Request, would be a breakthrough.

### Additional context

Here is the example of Typebot
![image](https://github.com/chatwoot/chatwoot/assets/47323533/c6e87a4b-6aa0-4308-a22c-06b8da3279cc)

And here, from the N8N
![image](https://github.com/chatwoot/chatwoot/assets/47323533/a814bbc6-138c-4a0b-a818-9e0f62f2924e)

**Comment by sanjayws** [comment]
Second this. It would be amazing.

**Comment by luizeof** [comment]
Please consider this feature ;)

**Comment by samyrwendel** [comment]
I completely agree with the previous comments! This feature would be fantastic and would greatly enhance our workflow efficiency. It would be amazing to see this implemented soon!

**Comment by FrancisMorelli** [comment]
Please consider this feature ;)

**Comment by lucianodltec** [comment]
+1

**Comment by mTondenier** [comment]
This feature would be incredible, instead of having to develop a huge set of macros in chatwoot, just adding this one would let users do whatever they want
